hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/snapshot/TestExportSnapshot.java

```diff
@@ -106,6 +106,7 @@ public static void setUpBaseConf(Configuration conf) {
     // If a single node has enough failures (default 3), resource manager will blacklist it.
     // With only 2 nodes and tests injecting faults, we don't want that.
     conf.setInt("mapreduce.job.maxtaskfailures.per.tracker", 100);
+    conf.setInt("snapshot.export.default.map.group", 1);
```

Comment:
Why we need this change?

Reply:
> Why we need this change?

The default value is 10, if the files to export are less than 10, there is only one mapper. The precondition to reproduce this issue is export a file in different mappers at the same time.
But it seems useless due to the file size is too small to reproduce this issue.